spec/factories/responses.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :response do
+    text "MyString"
+    created_at "2016-10-31 16:25:15"
```

Comment:
don't think i've had to put timestamps in factories before, is it okay without these?